lib/rules/landmark-at-least-one-main.json

```diff
@@ -0,0 +1,16 @@
+{
+  "id": "landmark-at-least-one-main",
+  "selector": "html",
+  "tags": [
+    "cat.keyboard"
```

Comment:
I agree with the best-practice, but cat.keyboard is what the current landmark check (used by the bypass rule) is categorized under